net-misc/ncp/files/ncp-1.2.4-fix-makefile.patch

```diff
@@ -0,0 +1,11 @@
+--- ncp-1.2.4/GNUmakefile	2005-05-13 19:17:17.000000000 +0200
```

Comment:
I usually like to put a [header](https://dev.gentoo.org/~vapier/clean-patches) with a Gentoo bug ref at least.

Reply:
ping

Reply:
I've updated the header, but didn't add a bug reference since i couldn't find one. Hope thats enough.